### PR TITLE
fixes #256 - displayed link to discussion page on main and details page

### LINKED ISSUE #256: Display link to discussion page on main page and store listing page
Display a link to loklak apps discussion page on the navbar of main page and details page, so that users can visit the discussion page easily

### PR DESCRIPTION
**Pull request by djmgit**
Fixes issue #256, displayed link to forum page on
the navbar of main and app details page.

I have:
- [x] There is a corresponding issue for this pull request.
- [x] Mentioned the Issue number in the pull request commit message `Fixes #<number> commit message`
- [x] There is only strictly only one commit per issue.

### For the reviewers
I have:
- [ ] Reviewed this pull request by an authorized contributor.
- [ ] The reviewer is assigned to the pull request.

### CODE REVIEW

**Comment by djmgit** [comment]
Test link: http://loklakapps.surge.sh/
@kavithaenair @SKrPl @singhpratyush @Achint08 open for review

File: cms/topmenu.json [hunk 1]
```diff
@@ -4,6 +4,7 @@
     {"Search": "http://loklak.net"},
     {"Blog": "http://blog.loklak.net/"},
     {"Developers": "http://dev.loklak.org"},
-    {"API": "http://api.loklak.org"}
+    {"API": "http://api.loklak.org"},
+    {"Discus": "/forum.html"}
```
Comment (Achint08) [inline cms/topmenu.json]: It is `Discuss`.
`Discus` is a fish. :P

Comment (djmgit) [inline cms/topmenu.json]: @Achint08 my bad :P

**Comment by djmgit** [comment]
@Achint08 made changes

**Comment by mariobehling** [comment]
Screenshot?

**Comment by mariobehling** [comment]
Not required.

**Pull request closed without merging**